release_notes/368-release-notes.adoc

```diff
@@ -29,6 +29,13 @@ For more details, see xref:../integration/integrate-with-image-registries.adoc#u
 [id="output-format-enhancements"]
 === Enhancements for CI outputs
 Red Hat has improved the usability of {product-title} CI integrations. CI outputs now show additional detailed information about the vulnerabilities and the security policies responsible for broken builds.
+For more details, see xref:../cli/getting-started-cli.adoc#configuring-output-format[Configuring output format].
+
+[id="automount-service-account-token-policy-criteria"]
+=== Automount Service Account Token policy criteria
+Kubernetes automatically provisions a service account during pod creation and mounts the account's secret token within the pod at runtime.
+Many containerized applications do not require direct access to the service account. If a threat actor compromises an application, they can glean for an account token to further compromise the server.
```

Comment:
Would extract be a better word to use in place of glean? Also, it looks like "they" is used here with a singular noun. 
If the threat actor here is referring to a person, disregard. 

Maybe:

> If a threat actor compromises an application, it can extract an account token to further compromise the server.

Reply:
Yes, the threat actor is referring to a human here. I replaced glean with `obtain`.